.php_cs.dist

```diff
@@ -0,0 +1,63 @@
+<?php
+
+$config = PhpCsFixer\Config::create();
+$config->getFinder()
+    ->exclude('vendor')
+    ->in(__DIR__);
+$config->setRules([
+    '@PSR1' => true,
```

Comment:
I dont mind using a defualt style like psr2 or symfony-style or similar.

I dont feel its worth having a „custom“ style... @evert any opinion on this?